fusillade-api.yml

```diff
@@ -994,11 +1111,52 @@ components:
         minimum: 5
         maximum: 30
         default: 15
+    policy_name:
+      name: policy_name
+      in: path
+      required: true
+      schema:
+        type: string
   schemas:
     paged_results:
       type: array
       items:
         type: string
+    resource_types:
```

Comment:
Should these all have `description`s?

Reply:
added some descriptions. We can add them overtime.